spec/controllers/protected_resources_controller_spec.rb

```diff
@@ -174,4 +174,101 @@ def doorkeeper_unauthorized_render_options(error: nil)
       end
     end
   end
+
+  context 'when custom forbidden render options are configured' do
+    after do
+      module ControllerActions
+        def doorkeeper_forbidden_render_options(error: nil)
+        end
+      end
+    end
+
+    controller do
+      before_filter -> { doorkeeper_authorize! :write }
+
+      include ControllerActions
+    end
+
+    let(:token_string) { '1A2DUWE' }
+
+    context 'with a JSON custom render' do
+      before do
+        module ControllerActions
+          def doorkeeper_forbidden_render_options(error: nil)
```

Comment:
Unused method argument - `error`. You can also write as `doorkeeper_forbidden_render_options(*)` if you want the method to accept any arguments but don't care about them.